source/auth.txt

```diff
@@ -185,32 +194,20 @@ To learn more, see the following:
 - :ref:`oidc-authentication-workload`
 - :ref:`set-up-pwdless-auth`
 
-X.509 Client Certificates 
-`````````````````````````
-
-|service| supports X.509 client certificates for database user
-authentication. You must follow best practices for password management
-and credential rotation. We recommend that you use complex passwords,
-automate credential rotations, or ideally, generate one-time passwords
-through a privileged access management (PAM) solution for each access.
-|service| integrates seamlessly with HashiCorp Vault, allowing secure
-management and automation of secrets and credentials for database
-access. 
-
-To learn more, see :manual:`x.509 </core/security-x.509/>`.
-
-SCRAM 
-`````
-
-|service| supports SCRAM for database user authentication. SCRAM is
-recommended only as a quick option for development or test environments.
-We recommend that you use complex passwords, automate credential
-rotations, or ideally, generate one-time passwords through a privileged
-access management (PAM) solution for each access. |service| integrates
-seamlessly with HashiCorp Vault, allowing secure management and
-automation of secrets and credentials for database access.
-
-To learn ore, see :manual:`SCRAM </core/security-scram/>`.
+X.509 Client Certificates and SCRAM 
+```````````````````````````````````
+
+The best practice is to leverage a 3rd party federated identity provider
+for accessing all aspects of the |service| control and data plane.
+|service| {+clusters+} do support :manual:`x.509
+</core/security-x.509/>` client certificates and :manual:`SCRAM
+</core/security-scram/>` for user authentication. However, use these
+only if you don't have an identity provider for federation. If you 
```

Comment:
I would break this into 2 paragraphs since they're separate ideas

```suggestion
only if you don't have an identity provider for federation. 

If you 
```